scripts/lay-down-os

```diff
@@ -16,7 +16,6 @@ do
 done
 
 BASE_DIR="/mnt/new_img"
-VERSION=${RANCHEROS_VERSION}
```

Comment:
It is pulled from the container environment variable. It is used in the file name of the initrd and vmlinuz so a user can see the versions by doing an ls in /boot if they needed to.